src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onSelectTab: (tab: Tab) => void;
+};
+
+export const Tabs:React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onSelectTab,
+}) => {
+  let selectedTabIndex = tabs.findIndex(tab => tab.id === selectedTabId);
+
+  selectedTabIndex = selectedTabIndex !== -1
+    ? selectedTabIndex
+    : 0;
```

Comment:
@mmahfuzi you can skip the index step - just apply the `.find` method
and then if it's not found, apply `tabs[0]`
```suggestion
  let selectedTab = tabs.find(tab => tab.id === selectedTabId)
    || tabs[0];
```